src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -239,6 +239,15 @@ All parameters are case-insensitive.</value>
 
 -WindowStyle | -w
     Sets the window style to Normal, Minimized, Maximized or Hidden.
+
+-SettingsFile | -settings
+    Overrides the system-wide powershell.config.json settings file for the session.
+    By default, system-wide settings are read from the powershell.config.json
+    in the $PSHOME directory.
+    Note that per-user settings take precedence over system-wide settings.
+
+    Example: pwsh -SettingsFile c:\myproject\powershell.config.json
```

Comment:
The scenario I think of is `pwsh -settingsfile blah.json -configurationname another-ps-instance-config-name`. In this case, shall the user expect the `blah.json` would be applied to the powershell instance that the `another-ps-instance-config-name` points to?

Maybe you can document the behavior in that case in the help content.

Reply:
I'll document it.

Reply:
Fixed